### ISSUE TITLE
TypeError: __str__ returned non-string (type bytes)

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#902]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/private/var/containers/Bundle/Application/F2D20355-4623-4D2D-938C-18912C041BF1/Electron-Cash.app/Library/Application Support/com.c3-soft.ElectronCash/app/ElectronCash/electroncash_gui/ios_native/utils.py", line 261, in run_with_except_hook
    run_original(*args2, **kwargs2)
  File "/var/containers/Bundle/Application/F2D20355-4623-4D2D-938C-18912C041BF1/Electron-Cash.app/Library/Python/lib/python35.zip/threading.py", line 862, in run
    self._target(*self._args, **self._kwargs)
  File "/private/var/containers/Bundle/Application/F2D20355-4623-4D2D-938C-18912C041BF1/Electron-Cash.app/Library/Application Support/com.c3-soft.ElectronCash/app/ElectronCash/electroncash/web.py", line 168, in get_payment_request_thread
    on_pr(request)
  File "/private/var/containers/Bundle/Application/F2D20355-4623-4D2D-938C-18912C041BF1/Electron-Cash.app/Library/Application Support/com.c3-soft.ElectronCash/app/ElectronCash/electroncash_gui/ios_native/gui.py", line 1022, in on_pr
    utils.NSLog("On PR: %s -- UNIMPLEMENTED.. IMPLEMENT ME!",str(request))

TypeError: __str__ returned non-string (type bytes)
```

Reporter
------------

This issue was reported by 1 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 3.3 (iOS Build: 301)  | 3.5.5+ (default, Feb 21 2018, 15:57:07)  [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)] | Darwin-18.0.0-iPhone8,1-64bit iOS 12.0 iPhone (639EF6F8-3C1F-4B27-94A2-4BD395FEB38E) | standard | en_UK | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by cculianu** [comment]
Some bug in dependent code but this call path is taken because I didn't finish implementing BIP70.

Will do that very soon (I hope!).

**Comment by cculianu** [comment]
This is fixed by #908
